package.json

```diff
@@ -5,15 +5,23 @@
   "dependencies": {
     "ajv": "6.5.4",
     "chart.js": "2.7.2",
+    "connected-react-router": "^4.5.0",
```

Comment:
Is this used?

Reply:
It doesn't appear to be. It's used so that you can control react router with redux. The integration is kind of awkward otherwise.